### PR TITLE
ID localization for pull-image-private-registry task.

### PR DESCRIPTION
**Pull request by ariscahyadi**
This is addressing translation for `/docs/tasks/configure-pod-container/pull-image-private-registry/` as mentioned in #20111

### CODE REVIEW

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+Kamu telah berhasil mengatur data pribadi untuk Docker kamu pada klaster sebagai sebuah Secret yang dipanggil dengan nama `regcred`.
+
+{{< note >}}
+
+Mengetik Secret pada baris perintah dapat menyimpannya dalam riwayat (_history_) dari _shell_ kamu tanpa perlindungan, dan
+Secret tersebut mungkin juga terlihat oleh pengguna lain dalam PC kamu selama perintah `kubectl` sedang berjalan.
+{{< /note >}}
+
+
+## Menginspeksi Secret `regcred`
```
Comment (girikuncoro) [inline content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md:122]: we can skip formatting on header now or add anchor helper
```suggestion
## Menginspeksi Secret `regcred` {#menginspeksi-secret-regcred}
```

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+wget -O my-private-reg-pod.yaml https://k8s.io/examples/pods/private-reg-pod.yaml
+```
+
+Dalam berkas `my-private-reg-pod.yaml`, ubah `<your-private-image>` dengan tautan ke _image_ dalam register pribadi seperti ini:
+
+```none
+your.private.registry.example.com/janedoe/jdoe-private:v1
+```
+
+Untuk menarik _image_ dari register pribadi, Kubernetes memerlukan data pribadi.
```
Comment (girikuncoro) [inline content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md:193]: this is nice! pull == menarik

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+kubectl apply -f my-private-reg-pod.yaml
+kubectl get pod private-reg
+```
+
+
+## {{% heading "whatsnext" %}}
+
+
+* Pelajari lebih lanjut tentang [Secret](/docs/concepts/configuration/secret/).
+* Pelajari lebih lanjut tentang [menggunakan register pribadi](id/docs/concepts/containers/images/#menggunakan-registri-privat).
```
Comment (girikuncoro) [inline content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md:208]: missing `/`
```suggestion
* Pelajari lebih lanjut tentang [menggunakan register pribadi](/id/docs/concepts/containers/images/#menggunakan-registri-privat).
```

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
+---
+title: Menarik Image dari Register Pribadi
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md:2]: Yes! and "register" is also an Indonesian word.

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+    --type=kubernetes.io/dockerconfigjson
+```
+
+Jika kamu memerlukan lebih banyak kontrol (misalnya, untuk mengatur _namespace_ atau label baru pada Secret) 
+maka kamu dapat menyesuaikan Secret tersebut sebelum menyimpannya.
+Pastikan untuk:
+
+
+- Mengatur nama dari pokok (_item_) data menjadi `.dockerconfigjson`
+- Melakukan enkode secara _base64_ dari berkas Docker (_Docker file_) dan memindahkan urutan huruf (_string_) tersebut, secara tidak terputus sebagai nilai untuk bidang `data[".dockerconfigjson"]`
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md:78]: I also just knew it :)

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+register atau repositori pribadi untuk Docker.
+
+
+## {{% heading "prerequisites" %}}
+
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Untuk melakukan latihan ini, kamu memerlukan sebuah
+[name pengguna (ID) Docker](https://docs.docker.com/docker-id/) dan kata kunci (_password_).
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Typo fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2
For "kata sandi", I agree with that, but not harm to put once as initial.

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+
+
+<!-- steps -->
+
+## Masuk (_log in_) ke Docker
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+    "auths": {
+        "https://index.docker.io/v1/": {
+            "auth": "c3R...zE2"
+        }
+    }
+}
+```
+
+{{< note >}}
+Jika kamu menggunakan tempat penyimpanan data pribadi (_credential_) untuk Docker, maka kamu tidak akan melihat entri `auth` tetapi entri `credsStore` dengan nama tempat penyimpanan sebagai nilainya.
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: I also just learned "kredensial" :)
Fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+
+Jika kamu sudah menjalankan `docker login`, kamu dapat menyalin data pribadi itu ke Kubernetes:
+
+```shell
+kubectl create secret generic regcred \
+    --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+    --type=kubernetes.io/dockerconfigjson
+```
+
+Jika kamu memerlukan lebih banyak kontrol (misalnya, untuk mengatur _namespace_ atau label baru pada Secret)
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Agree with that.
Fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+  name: myregistrykey
+  namespace: awesomeapps
+data:
+  .dockerconfigjson: UmVhbGx5IHJlYWxseSByZWVlZWVlZWVlZWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWxsbGxsbGxsbGxsbGxsbGxsbGxsbGxsbGxsbGxsbGx5eXl5eXl5eXl5eXl5eXl5eXl5eSBsbGxsbGxsbGxsbGxsbG9vb29vb29vb29vb29vb29vb29vb29vb29vb25ubm5ubm5ubm5ubm5ubm5ubm5ubm5ubmdnZ2dnZ2dnZ2dnZ2dnZ2dnZ2cgYXV0aCBrZXlzCg==
+type: kubernetes.io/dockerconfigjson
+```
+
+Jika kamu mendapat pesan kesalahan `error: no objects passed to create`, ini berarti pengkodean _base64_ dari urutan huruf tersebut tidak valid.
+Jika kamu mendapat pesan kesalahan seperti `Secret "myregistrykey" is invalid: data[.dockerconfigjson]: invalid value ...`, ini berarti
+enkode _base64_ dari urutan huruf dalam data tersebut sukses didekodekan, tetapi tidak bisa di uraikan menjadi berkas `.docker/config.json`.
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+
+Buatlah Secret ini, dan berilah nama `regcred`:
+
+```shell
+kubectl create secret docker-registry regcred --docker-server=<your-registry-server> --docker-username=<your-name> --docker-password=<your-pword> --docker-email=<your-email>
+```
+
+dimana:
+
+* `<your-registry-server>` merupakan FQDN dari register private Docker kamu. (https://index.docker.io/v1/ untuk DockerHub)
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 12]
```diff
@@ -0,0 +1,212 @@
...
+* `<your-name>` adalah nama pengguna Docker kamu.
+* `<your-pword>` adalah kata kunci Docker kamu.
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md (hunk 1 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Agree! Fixed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2

File: content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md (hunk 3 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md]: FIxed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2

**Comment by ariscahyadi** [comment]
> great work, beautifully localized @ariscahyadi ! 🎉
> I learned new indonesian words:
> encode -> http://kbbi.web.id/enkode
> decode -> http://kbbi.web.id/dekode
> pull image -> tarik _image_
> 
> just couple of nits before we get this in

Thanks, @girikuncoro !
Let's learn new Indonesian words together :)

**Comment by girikuncoro** [comment]
@ariscahyadi thanks! let's continue the great work for next milestone :)

/lgtm
/approve

**Pull request merged**